packages/primitives/src/Card.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import { createComponent } from '@napred/ds';
+
+const Card = createComponent('Card', 'div');
+export const CardHeader = createComponent('CardHeader', 'div');
+export const CardFooter = createComponent('CardFooter', 'div');
+
+// $FlowExpectError
+Card.defaultProps = {
+  flexDirection: 'column',
```

Comment:
No, actually, we will remove the flex stuff from Card completely. Card is just a structured div tree, which can be flexed, but by default it is not. The flexDirection was there just to secure that when user makes display=flex, the card will be turned into column and not a row. It is wrong to do this automatically by user, because we cannot assume what user wants to achieve. We will delegate this responsibility to the user of Card, so for example ProductCard will be not only flexed, but also flexDirection=column. Also we don't need to do flexing for card subcomponents, this can be also delegated to the user.